internal/pkg/deploy/pipeline_test.go

```diff
@@ -201,7 +202,7 @@ func TestPipelineSourceFromManifest(t *testing.T) {
 
 func TestPipelineBuild_Init(t *testing.T) {
 	const (
-		defaultImage   = "aws/codebuild/amazonlinux2-x86_64-standard:3.0"
```

Comment:
Are we required to update this var if we update the real default image version? (I guess not?)

If not, what do you think of we  just use a value like "mock/image:1.0" so that it doesn't appear in search results 🤔

Reply:
:rip: sadly it looks like we need to use the real default image because this test checks that the `image` used by default is indeed "4.0"